environment/metadata.go

```diff
@@ -47,6 +49,10 @@ func registerComputeType(dataString *MetaDataStrings) {
 func registerBucket(dataString *MetaDataStrings) {
 	flag.StringVar(&(dataString.Bucket), "bucket", "", "s3 bucket ex cloudwatch-agent-integration-bucket")
 }
+func registerS3Key(dataString *MetaDataStrings) {
+	flag.StringVar(&(dataString.S3Key), "s3key", "release/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm",
```

Comment:
default value for the flag is the s3key for the RPM. DEB and MSI testing can override it in the GitHub Action code in the same way that the s3 bucket is passed in.